VocaDbModel/Mapping/Security/PermissionTokenMap.cs

```diff
@@ -1,18 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using FluentNHibernate.Mapping;
-using VocaDb.Model.Domain.Security;
-
-namespace VocaDb.Model.Mapping.Security {
-
-	public class PermissionTokenMap : ClassMap<PermissionToken> {
-
-		public PermissionTokenMap() {
-
-			Id(m => m.Id);
-
-		}
-	}
-}
```

Comment:
I removed PermissionTokenMap.cs because it seems that it is not used. In fact, if it is included in the project, an NHibernate.InvalidProxyTypeException will be thrown.